common/djangoapps/student/models.py

```diff
@@ -336,9 +354,18 @@ def requires_parental_consent(self, date=None, age_limit=None, default_requires_
         year_of_birth = self.year_of_birth
         if year_of_birth is None:
             return default_requires_consent
+
         if date is None:
-            date = datetime.now(UTC)
-        return date.year - year_of_birth <= age_limit
+            age = self.age
+        else:
+            age = date.year - year_of_birth
+
+        return age <= age_limit
+
+    def __enumerable_to_display(self, enumerables=(), enum_value=None):
```

Comment:
Do these parameters need to be optional? I generally try to avoid functions which either return a value or `None`, and requiring the params would prevent this.